_conferences/mdevcamp_2018.md

```diff
@@ -0,0 +1,10 @@
+name: "mDevCamp"
+website: https://mdevcamp.eu/
+location: Prague, Czechia
+
+date_start: 2018-6-15
+date_end:   2018-6-15
+
+cfp_start: 2018-1-8
```

Comment:
Shouldn't those be prefixed with a leading zero?

Reply:
It probably doesn't matter. Depends on how strict of a validating parser Jekyll has and it appears (from the CI build) to be lenient.